.github/workflows/tests.yml

```
@@ -1,51 +1,38 @@
 name: continuous-integration
 
-on: [push, pull_request]
+on:
+  push:
+    branches: [master]
```

```suggestion
    branches: [master]
    tags:
      - 'v*'
```

otherwise pypi releases aren't made!